website/docs/intro/phases/collaborate.mdx

```diff
@@ -28,6 +28,8 @@ As your team grows, you may run into the risk of concurrent operations on state
 
 [Get started with HCP Terraform](/terraform/tutorials/cloud-get-started) and learn how to [securely store your Terraform state](/well-architected-framework/security/security-sensitive-data#storing-terraform-state).
 
+To migrate your existing state to HCP Terraform, refer to the [Migrate Terraform state to HCP Terraform](/terraform/cloud-docs/migrate) documentation. To migrate learn how to migrate Terraform state files to HCP Terraform in bulk, complete the [Migrate to HCP Terraform in bulk](/terraform/tutorials/cloud/bulk-migrate-hcp) tutorial and refer to the [Terraform migrate documentation](/terraform/cloud-docs/migrate/tf-migrate).
```

Comment:
@BrianMMcClain Have you tried this as a list? I think it'd be easier to read.

Reply:
Good call! Updated in the latest commit.